### PR TITLE
Form item brief_text, info_group, and updating form structure

### PR DESCRIPTION
**Pull request by gigxz**
Addressing forms feedback from Ash. This is up on QA currently.

### CODE REVIEW

File: drivers/hmis/app/graphql/schema.graphql [hunk 1]
```diff
@@ -367,6 +367,11 @@ enum Component {
   """
   HORIZONTAL_GROUP
 
+  """
+  Render contents in an info box
+  """
+  INFO_GROUP
```
Comment (gigxz) [inline drivers/hmis/app/graphql/schema.graphql:373]: This is sort of TBD, it's something Ash is toying with for displaying data quality inputs in visually distinct boxes. Once we decide if/how we're going to use it, maybe I can come up with a better name.

File: drivers/hmis/app/graphql/schema.graphql [hunk 2]
```diff
@@ -1300,6 +1305,11 @@ type FormItem {
   autofillValues: [AutofillValue!]
   bounds: [ValueBound!]
 
+  """
+  Label to use for placeholder and population table
+  """
+  briefText: String
```
Comment (gigxz) [inline drivers/hmis/app/graphql/schema.graphql:1311]: Select fields should have placeholders (`Select <x>...`). The `briefText` field gets used for that, if it exists.

It also gets used in the record picker dialog, so the table can show "Homeless on previous night" while the form asks "On the night before, did the client stay on the streets, ES or SH?"

File: drivers/hmis/app/graphql/types/forms/pick_list_option.rb [hunk 3]
```diff
@@ -35,7 +35,7 @@ def self.options_for_type(pick_list_type, user:, project_id: nil)
         state_options.map do |obj|
           {
             code: obj['abbreviation'],
-            label: "#{obj['abbreviation']} - #{obj['name']}",
+            # label: "#{obj['abbreviation']} - #{obj['name']}",
```
Comment (gigxz) [inline drivers/hmis/app/graphql/types/forms/pick_list_option.rb:38]: Switching state picker to a small width input with only abbreviations in the dropdown

File: drivers/hmis/lib/form_data/assessments/base_assessment.json [hunk 4]
```diff
@@ -1592,6 +1601,7 @@
         },
         {
           "type": "CHOICE",
+          "component": "RADIO_BUTTONS",
```
Comment (gigxz) [inline drivers/hmis/lib/form_data/assessments/base_assessment.json:1604]: Changed default behavior so `NoYesReasonsForMissingData` doesn't default to radio buttons. Need to specify them explicitly.

Radio buttons are defaulted if there are fewer than 4 options.

Comment (gigxz) [reply drivers/hmis/lib/form_data/assessments/base_assessment.json:1604]: We should probably fill in `COMPONENT` for everything, so we don't have to change these when the default changes

**Review by eanders — APPROVED** [review]

**Comment by eanders** [comment]
Looks good to me!  Thanks.

**Pull request merged**